src/modules/update.module.js

```diff
@@ -0,0 +1,12 @@
+/* eslint "import/prefer-default-export":"off" */
+import { readLocalStorage, writeLocalStorage } from './local-storage.module.js';
+
+function update(item) {
+  const items = readLocalStorage();
+  const element = items.filter((i) => i.id === item.id)[0];
+  element.description = item.description;
+  items[item.id - 1] = element;
+  writeLocalStorage(items);
```

Comment:
- [ ] Kindly add a function to edit task descriptions

Reply:
the update function edits the task description @uwadonat